### DISCUSSION TITLE
What happens to changes under review?

### DISCUSSION THREAD

**Discussion by TheAdventurer64** [organicmaps/organicmaps#2949] [Q&A] [answered]
After saving some changes using the POI editor, I got this message saying that the changes would be reviewed:

![image](https://user-images.githubusercontent.com/50059322/178365819-b11fb9ed-56e1-4763-9714-2ad6fe3b4df5.png)

So what does this mean? Because (from my knowledge) OM doesn't review individual map changes, will the changes be uploaded or will they be sent to a review center where they won't be reviewed?

**Comment by biodranik** [comment, marked as answer]
Everything is uploaded to OpenStreetMap immediately (except if there is some bug in our code), and can be reviewed by OSM community later. You can check the status of edits by typing ?edits in the search. The text was made as a matter of protection against vandalism from many users.

**Reply by TheAdventurer64** [reply]
How long does it take changes to be uploaded? I checked the POI out online, and it doesn't appear to have been changed.

**Reply by biodranik** [reply]
Usually when the app goes into the background. Type ?edits in the search to see upload status.

**Reply by TheAdventurer64** [reply]
It's been over a day since my edit was made and it hasn't be uploaded yet. Is it because I made the edit in Airplane Mode?

**Reply by biodranik** [reply]
There could be also a conflict or a bug in our code. That's why it is important to check what's the status of the edit in the app.

**Reply by TheAdventurer64** [reply]
> There could be also a conflict or a bug in our code. That's why it is important to check what's the status of the edit in the app.

The status bar says "Uploaded: 0." Does that help?

**Reply by biodranik** [reply]
And nothing else? Did you save your edits?

**Reply by TheAdventurer64** [reply]
> And nothing else? Did you save your edits?

The screen also shows the edit which  was saved.

**Reply by biodranik** [reply]
Then it means the background upload service can't connect to the internet. You can enable logs recording or check logs with adb logcat while activating and deactivating the app, and check it for any upload/connection errors (or share with us to check).

**Reply by TheAdventurer64** [reply]
> Then it means the background upload service can't connect to the internet. You can enable logs recording or check logs with adb logcat while activating and deactivating the app, and check it for any upload/connection errors (or share with us to check).

The problem with that is that I use an iOS device.

**Reply by biodranik** [reply]
That's interesting. Did you try to edit something else? Is your osm.org account history empty?

**Reply by csidirop** [reply]
I had recent edits pushed to osm without errors.

![grafik](https://user-images.githubusercontent.com/43964592/179397479-638571e2-9eda-433e-bd37-5e2493a6ffb3.png)

**Reply by TheAdventurer64** [reply]
> That's interesting. Did you try to edit something else? Is your osm.org account history empty?

I figured it out. Apparently, I made the changes without logging in. Maybe editing offline doesn't trigger the requirement to login in order to edit.

**Reply by rtsisyk** [reply]
@TheAdventurer64 , have we answered your question here? if yes, could you please mark it as answered?
